openhsi/cameras.py

```diff
@@ -27,26 +25,33 @@ def __init__(self, mode:str = None, **kwargs):
         """Initialise Webcam"""
         super().__init__(**kwargs)
 
+        try:
+            import cv2
+        except ModuleNotFoundError:
+            warnings.warn("ModuleNotFoundError: No module named 'cv2'. Did you try `pip install opencv-python`?",stacklevel=2)
```

Comment:
do not think we should assume pip.

Reply:
ok, maybe remove the Did you try bit? I've seen it used in other libraries. Or add the equivalent conda command as well?

Reply:
maybe we go generic?

> Did you have `opencv-python` installed?